siliconcompiler/utils.py

```diff
@@ -239,3 +240,9 @@ def default_credentials_file():
     cfg_file = os.path.join(Path.home(), '.sc', 'credentials')
 
     return cfg_file
+
+
+def register_sc_data_source(chip):
+    chip.register_package_source('siliconcompiler_data',
+                                 'git+https://github.com/siliconcompiler/siliconcompiler',
+                                 'v'+sc_version)
```

Comment:
This shouldn't matter much because we don't plan on changing the PDKs and LIBs,
but if we do want to edit them it might be confusing to have to do a release first.

Reply:
this will eventually become one of the data repos. Using SC is just a temporary solution until they are ready.